vadim.martsun/task_4/src/main/java/random/RandomBankAccountService.java

```diff
@@ -0,0 +1,10 @@
+package random;
+
+import objects.Account;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
```

Comment:
You don't need to use @Service on interface. It's enough to have in on implementation